packages/launchpad-navigator/src/floating-navigator/style.scss

```diff
@@ -4,37 +4,7 @@
 	position: fixed;
 	top: calc(var(--masterbar-height) + 2px);
 	right: 0.5em;
-	background-color: #fff;
-	color: #000;
-	z-index: 9999;
 	cursor: default;
 	transition: max-height 0.5s;
 	min-width: 25em;
-	padding-top: 16px;
-
-	.launchpad-navigator__floating-navigator-header {
-		display: flex;
-		align-items: center;
-		justify-content: space-between;
-
-		h2 {
-			font-family: $font-sf-pro-display;
-			font-size: $font-title-small;
-			font-weight: 500;
```

Comment:
I think we still need these values to be specified as an override for the underlying component. In my case, I am not seeing the right font declaration (mainly because it's missing the `"SF Pro Display"` value), and the font-size is being computed as 19.2px / 1.2rem instead of 20px / 1.25rem, and the `font-weight` is 600 instead of 500.

I think you can handle that as follows:
```
.components-modal__header .components-modal__header-heading {
	font-family: $font-sf-pro-display;
	font-size: $font-title-small;
	font-weight: 500;
}
```